### ISSUE TITLE
Mehr Infos hier

### ISSUE THREAD

**Issue by Dtrieb** [haxko/ctr30-manual#1]
Guck mal was ich noch gefunden habe:
https://www.wak-lab.org/3d-led-weihnachtsbaum/

**Comment by adlerweb** [comment]
Ja, mehrere Anbieter stellen verschiedene Anleitungen bereit. Die Verlinkte nutzt leider nur 4-Ring-Widerstände (in unseren Bausätzen war teils 4-Ring und 5-Ring gemischt). Leider ist zudem keine Lizenz angegeben, daher ist es ohne Einzelgenehmigung vermutlich auch nicht erlaubt diese zu ändern oder im Rahmen von Veranstaltungen zu kopieren und weiterzugeben.
